_data/faq.yml

```
@@ -224,13 +224,26 @@
 
 - category_name: Pricing
   items:
+    - question: What payment options do you accept?
+      answer: |
+        <p>
+          We accept AWS Marketplace for annual and multi-year contracts. For monthly billing, we require automated payments via credit card or ACH transfer. If you wish to use invoices and pay by wire transfer or check instead, you must do annual billing.
```

Might want to make it more clear that AWS Marketplace is _an_ option but not necessarily the first or primary option.

```suggestion
          For monthly billing, we require automated payments via credit card or ACH transfer. If you wish to use invoices and pay by wire transfer or check instead, you must do annual billing. We also accept AWS Marketplace for annual contracts.
```